modules/consensus/subscribe.go

```diff
@@ -149,41 +104,36 @@ func (cs *ConsensusSet) updateSubscribers(revertedNodes []*blockNode, appliedNod
 	cs.changeLog = append(cs.changeLog, ce)
 
 	// Notify each update channel that a new update is ready.
-	for _, subscriber := range cs.subscriptions {
-		// If the channel is already full, don't block.
-		select {
-		case subscriber <- struct{}{}:
-		default:
-		}
+	cc, err := cs.computeConsensusChange(len(cs.changeLog) - 1)
+	if err != nil && build.DEBUG {
+		panic(err)
+	}
+	for _, subscriber := range cs.subscribers {
+		subscriber.ReceiveConsensusSetUpdate(cc)
```

Comment:
perhaps in light of the new subscription model this should be called `ProcessConsensusChange`?